rpmlb/builder/koji.py

```diff
@@ -44,7 +44,10 @@ def __init__(
         self.epel = koji_epel
 
         #: Format string for the build target
-        self.target_format = koji_target_format
+        if koji_target_format is None:
+            self.target_format = type(self).DEFAULT_TARGET_FORMAT
```

Comment:
Ah, yes, this is probably what I heard and remembered it wrongly… I will get rid of the `type(…)` wrap.